x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/popup_options_to_versions.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
```

Comment:
I wonder if we should just have this file be the place we store all of the version information for protections.

We'd eventually add Ransomware and Register as AV.  So we could rename this to just be `policy_options_to_versions.ts`

@paul-tavares @efreeti let me know your thoughts